### PR TITLE
Bugfix/bookmark page nil error

### PR DESCRIPTION
**Pull request by nbfalcon**
I had this error while using `DOOM` Emacs; it is because your `null` check is wrongly ordered. See my commit messages for further details.

### CODE REVIEW

**Comment by nicolaisingh** [comment]
Thanks for this.  Can I ask how one can reproduce this error?

**Review by nicolaisingh — DISMISSED** [review]
Requesting to revert this change.  That code is intentionally like that to make it similar with the code of`saveplace`.

File: saveplace-pdf-view.el [hunk 1]
```diff
@@ -69,10 +69,8 @@
                                (or (null (cdr page))
                                    (= 1 (cdr page)))
                                (or (equal '(0.0 . 0.0) (cdr origin))
-                                   (eq nil (cdr origin))))))
-            (setq save-place-alist
-                  (cons (cons item `((pdf-view-bookmark . ,bookmark)))
-                        save-place-alist))))))))
+                                   (null (cdr origin))))))
+            (push `(,item (pdf-view-bookmark . ,bookmark)) save-place-alist)))))))
```
Comment (nicolaisingh) [inline saveplace-pdf-view.el]: Requesting to revert this change. That code is intentionally like that to make it similar with the code of `saveplace`.

Comment (nicolaisingh) [reply saveplace-pdf-view.el]: Please keep the change on the line `(null (cdr origin ))...`

**Comment by nbfalcon** [comment]
For me, this error is consistently triggered while restarting Emacs and when restoring the session (`SPC qr` in DOOM).

**Review by nicolaisingh — APPROVED** [review]

File: saveplace-pdf-view.el (hunk 1 above)
Comment (nbfalcon) [reply saveplace-pdf-view.el]: Fixed; that was bad commit chunking on my part, sorry. Should I make another PR?
